app/controllers/kudos_controller.rb

```diff
@@ -57,6 +57,14 @@ def create
         end
       end
     end
+  rescue ActiveRecord::RecordNotUnique
```

Comment:
I like keeping the `rescue` here rather than having it apply to all actions like it does in the `UsersController`. 👍

Reply:
Yeah same. Now I'll look for a reason to edit the user controller and smuggle in a similar change.

Reply:
[AO3-5677](https://otwarchive.atlassian.net/browse/AO3-5677) seems like a good choice for that, if you're looking for an excuse to fix it.